lib/auto_api/state.ex

```diff
@@ -118,6 +119,15 @@ defmodule AutoApi.State do
         def parse_state_property(:properties, []) do
```

Comment:
Shouldn't this and the following methods be written before line 102? Otherwise I don't think it can be called as `parse_state_property(_, [])` matches first.
It works now as they have the same output..

Reply:
Changed it to 
```
        def parse_state_property(:properties, _) do
          <<>>	       
        end
```